.github/workflows/pull-request-labeler.yml

```diff
@@ -0,0 +1,26 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+name: Pull Request Labeler
+
+on:
+  pull_request_target:
+    types: [opened, reopened, synchronize, converted_to_draft, ready_for_review, review_requested]
+    branches:
+      - main
+  pull_request_review:
+    types: [submitted]
+    branches:
+      - main
+
+jobs:
+  pull_request_labeler:
+    runs-on: ubuntu-latest
+    if: github.actor != 'github-actions[bot]' && github.actor != 'dependabot[bot]' && github.actor != 'MickeyMoz'
+    steps:
+      - name: Pull Request Labeler
+        uses: gabrielluong/pull-request-labeler@1.0.1
```

Comment:
Should we change the repo [firefox-android](https://github.com/mozilla-mobile/firefox-android)?

Reply:
Probably not worthwhile with the monorepo migration.